### PR TITLE
[Darkside] Chat CSS update

### PR DESCRIPTION
**Pull request by KenAJoh**
### Component Checklist 📝

- [ ] JSDoc
- [ ] Examples
- [ ] Documentation / Decision Records
- [ ] Storybook
- [ ] Style mappings (`@navikt/core/css/config/_mappings.js`)
- [ ] Component tokens (`@navikt/core/css/tokens.json`)
- [ ] CSS class deprecations (`@navikt/aksel-stylelint/src/deprecations.ts`)
- [ ] Exports (`@navikt/core/react/src/index.ts` and `@navikt/core/react/package.json`)
- [ ] New component? CSS import (`@navikt/core/css/index.css`)
- [ ] Breaking change? Update migration guide. Consider codemod.
- [ ] Changeset (Format: `<Component>: <gitmoji?> <Text>.` E.g. "Button: :sparkles: Add feature xyz.")

### CODE REVIEW

File: @navikt/core/css/darkside/chat.darkside.css [hunk 1]
```diff
@@ -1,33 +1,30 @@
...
 
-.navds-chat__bubble-wrapper {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-  display: flex;
-  flex-direction: column;
-  gap: var(--a-spacing-3);
-}
+  .navds-chat__bubble-wrapper {
```
Comment (HalvorHaugan) [inline @navikt/core/css/darkside/chat.darkside.css]: Should we discuss whether or not to use `&` when it's optional? I think I prefer always using it, b.c. it makes it easier to see that it's a nested selector. It's mostly a problem in GitHub though. It's quite easy to see in vscode b.c. of the vertical "rulers".

File: @navikt/core/react/src/chat/chat.stories.tsx [hunk 2]
```diff
@@ -88,8 +88,8 @@ export const Variants: Story = {
         <React.Fragment key={variant}>
           <h3>{variant}</h3>
           <Chat
-            avatar="NAV"
-            name="NAV"
+            avatar="Ls"
+            name="Ls"
```
Comment (HalvorHaugan) [inline @navikt/core/react/src/chat/chat.stories.tsx:92]: Any special reason for changing this? Just curious

Comment (KenAJoh) [reply @navikt/core/react/src/chat/chat.stories.tsx:92]: Just since it was used in Figma

File: @navikt/core/css/darkside/chat.darkside.css (hunk 1 above)
Comment (KenAJoh) [reply @navikt/core/css/darkside/chat.darkside.css]: Makes sense 👍 Just to confirm, in this case you would expect this syntax:
```css
& .navds-chat__bubble-wrapper {
    ...
  }
```

Comment (HalvorHaugan) [reply @navikt/core/css/darkside/chat.darkside.css]: Exactly!

**Review by HalvorHaugan — APPROVED** [review]

**Pull request merged**